src/components/KeycloakStatusPage/KeycloakStatusPage.tsx

```diff
@@ -0,0 +1,88 @@
+import { MESSAGE_PNC_ADMIN_CONTACT, MESSAGE_WAIT_AND_REFRESH } from 'common/constants';
+
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { ErrorPage } from 'components/ErrorPage/ErrorPage';
+import { PageLayout } from 'components/PageLayout/PageLayout';
+
+import { keycloakService } from 'services/keycloakService';
+import * as webConfigService from 'services/webConfigService';
+
+const webConfig = webConfigService.getWebConfig();
+
+interface IKeycloakStatusPageProps {
+  // Title will be used when constructing page title and page will be presented as error page, if undefined, standard page will be rendered.
```

Comment:
`If undefined, Keycloak status page will be rendered. Otherwise, the title will be used as the title of the error page that is rendered instead.`